pkg/codegen/templates/fiber/fiber-middleware.tmpl

```diff
@@ -27,7 +27,7 @@ func (siw *ServerInterfaceWrapper) {{$opid}}(c *fiber.Ctx) error {
   }
   {{end}}
   {{if .IsStyled}}
-  err = runtime.BindStyledParameter("{{.Style}}",{{.Explode}}, "{{.ParamName}}", c.Params("{{.ParamName}}"), &{{$varName}})
+  err = runtime.BindStyledParameter("{{.Style}}", {{.Explode}}, {{.Required}}, "{{.ParamName}}", c.Params("{{.ParamName}}"), &{{$varName}})
```

Comment:
I think this will fail now, as we've introduced [a new function `BindStyledParameterWithOptions`](https://github.com/oapi-codegen/runtime/blob/v1.1.0/bindparam.go#L71) - mind refactoring these templates to use that new function now?